### PR TITLE
DDCE-1746 content changes and tests fix

### PR DESCRIPTION
**Pull request by kuldeepwadhwa9**
# DDCE-????

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [ ]  I've updated the README if there are functionality changes
- [ ]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [ ]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [ ]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [ ]  I've squashed my commits - including the JIRA issue number in the commit message
- [ ]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -26,8 +26,7 @@ service.lang.cymraeg = Cymraeg
 Service.continue = Yn eich blaen
 Service.confirm = Cadarnhau
 Service.confirmAndContinue = Cadarnhau ac yn eich blaen
-Service.register.single = Cofrestru’r buddiant hwn
-Service.register.multiple = Cofrestru’r buddiannau hyn
+Service.register.confirm = Cadarnhau ac yn eich blaen
```
Comment (anburoach) [inline conf/messages.cy:29]: same as above

File: conf/messages [hunk 2]
```diff
@@ -127,15 +126,12 @@ AddBenefits.CY.Caution.p12.3 = (starting 6 April {0})
 
 AddBenefits.noselection.error = You need to select at least 1 benefit or expense before you can continue
 
-AddBenefits.Confirm.Single.Title = Register a benefit or expense
-AddBenefits.Confirm.Multiple.Title = Register benefits and expenses
+AddBenefits.Confirm.Single.Title = Check the benefits and expenses you want to register
```
Comment (anburoach) [inline conf/messages:129]: I don't see use of the key `AddBenefits.Confirm.Single.Title` anywhere in the code so remove from here

File: conf/messages.cy [hunk 3]
```diff
@@ -120,17 +119,14 @@ AddBenefits.CY.Caution.p12.3 = (sy’n dechrau ar 6 Ebrill {0})
 AddBenefits.noselection.error = Mae’n rhaid i chi ddewis o leiaf 1 buddiant neu draul cyn i chi allu mynd yn eich blaen
 
 AddBenefits.Confirm.Single.Title = Cofrestru buddiant neu draul
```
Comment (anburoach) [inline conf/messages.cy:121]: I don't see use of the key `AddBenefits.Confirm.Single.Title` anywhere in the code so remove from here

File: conf/messages.cy (hunk 1 above)
Comment (kuldeepwadhwa9) [reply conf/messages.cy]: Done

File: conf/messages.cy (hunk 3 above)
Comment (kuldeepwadhwa9) [reply conf/messages.cy]: Done

File: test/views/ConfirmNextYearViewSpec.scala [hunk 4]
```diff
@@ -54,8 +54,6 @@ class ConfirmNextYearViewSpec extends PBIKViewSpec {
       messages("Overview.next.heading", taxYearRange.cy + "", taxYearRange.cyplus1 + "")
         + " " + messages("AddBenefits.Confirm.Single.Title")
```
[the remark this answers is not in the document]
Comment (kuldeepwadhwa9) [reply test/views/ConfirmNextYearViewSpec.scala]: Done

**Review by anburoach — APPROVED** [review]
Approved

**Pull request merged**